azurelinuxagent/common/conf.py

```diff
@@ -502,6 +503,14 @@ def get_monitor_network_configuration_changes(conf=__conf__):
     return conf.get_switch("Monitor.NetworkConfigurationChanges", False)
 
 
+def get_ga_updates_enabled(conf=__conf__):
+    """
+    If True, the agent go through update logic to look for new agents otherwise it will stop agent updates.
+    NOTE: This option is needed in e2e tests to control agent updates.
```

Comment:
If only needed by the tests, this should be a Debug/experimental option

Reply:
@nagworld9 - did we decide this to expose this option to the user? if yes, let's add it to README, if not, let's make it Debug

Reply:
sorry, what is the difference between GAUpdates.Enabled and Debug.EnableGAVersioning

Reply:
No, this is not exposed to the user. I'll change to debug. 

Debug.EnableGAVersioning- This flag is like kill switch. Once this is trun off our code will go to self update path.

GAUpdates.Enabled- This flag is to stop the agent updates. Only used in e2e tests.

Reply:
I'll change as part of next work item.